### ISSUE TITLE
Accessibility bug: options menu aria-label is sub-optimal

### ISSUE THREAD

**Issue by SleeplessByte** [bluesky-social/social-app#6311]
### Steps to Reproduce

1. Go to https://bsky.app
2. Use a screenreader and tab onto a more options button for a post

```text
clickable  Open post options menu  menu button  collapsed  subMenu    link  
```

And then the first time you activate a menu item (arrow up/down)

```text
Open post options menu menu item <whatever the item is>
```

When a `<button>` has an `aria-label`, it should not have the word `button` inside the label, as the screenreader will add that for us (this does not have that), but in the same way, when there is `aria-haspopup="menu"`, the fact that there is a menu will be added. Together with `aria-expanded="false"`, there is ample information to detect that clicking this button will *open the post options menu*.

A better label would be:

```
Post options
```

The text would then be:
```
clickable  Post options  menu button  collapsed  subMenu    link  
```

and

```
Post options menu item <whatever the item is>
```

### What platform(s) does this occur on?

Web (Desktop), Web (Mobile)

### Device Info

Not relevant

### What version of the app are you using?

Web

### Additional Information

I strongly recommend there is a guidance document or a review step that checks these.
